tmt/steps/finish/__init__.py

```diff
@@ -17,6 +17,10 @@ class Finish(tmt.steps.Step):
     successful) so that provisioned systems are not kept running.
     """
 
+    def __init__(self, data, plan):
+        """ Initialize prepare step data """
+        super().__init__(data, plan)
```

Comment:
Why is this necessary if it only executes parent's init?

Reply:
removed